qCC/ccColorFromScalarDlg.h

```diff
@@ -0,0 +1,111 @@
+//##########################################################################
+//#                                                                        #
+//#                              CLOUDCOMPARE                              #
+//#                                                                        #
+//#  This program is free software; you can redistribute it and/or modify  #
+//#  it under the terms of the GNU General Public License as published by  #
+//#  the Free Software Foundation; version 2 or later of the License.      #
+//#                                                                        #
+//#  This program is distributed in the hope that it will be useful,       #
+//#  but WITHOUT ANY WARRANTY; without even the implied warranty of        #
+//#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the          #
+//#  GNU General Public License for more details.                          #
+//#                                                                        #
+//#          COPYRIGHT: CloudCompare project                               #
+//#                                                                        #
+//##########################################################################
+
+#ifndef CC_FROM_SCALAR_DLG_HEADER
+#define CC_FROM_SCALAR_DLG_HEADER
+
+//Qt
+#include <QColor>
+
+#include <ui_colorFromScalarDlg.h>
+#include <ccColorScale.h>
+#include <ccScalarField.h>
+#include <ccHistogramWindow.h>
+
+class ccPointCloud;
+
+//! Dialog to change the color levels
+class ccColorFromScalarDlg : public QDialog, public Ui::ColorFromScalarDialog
```

Comment:
UI classes shouldn't be done with inheritance.

They should be [pointer members](https://doc.qt.io/qt-5.9/designer-using-a-ui-file.html#using-a-pointer-member-variable).